nixos/modules/services/x11/redshift.nix

```diff
@@ -18,26 +19,6 @@ in {
       '';
     };
 
-    latitude = mkOption {
-      type = types.nullOr types.str;
-      default = null;
-      description = ''
-        Your current latitude, between
-        <literal>-90.0</literal> and <literal>90.0</literal>. Must be provided
-        along with longitude.
-      '';
-    };
-
-    longitude = mkOption {
```

Comment:
You need to add two `mkAliasOptionModule` so people don't get errors with this change.